### PR TITLE
fix(pacmak): label "External" stability as "Stable"

### PR DESCRIPTION
**Pull request by rix0rrr**
The distinction between "Stable" (checked stability) and "External"
(unchecked stability) is lost on users, and honestly we shouldn't bother
them with it.

Treat them the same when generating code.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/IStableInterfaceProxy.cs [hunk 1]
```diff
@@ -4,19 +4,13 @@
 
 namespace Amazon.JSII.Tests.CalculatorNamespace
 {
-    /// <remarks>
-    /// <strong>Stability</strong>: Stable
-    /// </remarks>
```
Comment (eladb) [inline packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/IStableInterfaceProxy.cs:9]: Why is this now omitted? Sounds like a useful thing to put in the docs, no?

Comment (rix0rrr) [reply packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/IStableInterfaceProxy.cs:9]: It actually makes more sense to me to do the opposite: stable unless indicated otherwise. We already did that for Python and it makes sense.

It's also how every other API is documented. Nobody else just slathers on "stable" everywhere, they *might* note "deprected" or "experimental" as it applies.

File: packages/jsii/test/negatives/neg.submodules-cannot-share-symbols.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-///!MATCH_ERROR: Symbol is re-exported under two distinct submodules (ns1 and ns2)
+///!MATCH_ERROR: Symbol is re-exported under two distinct submodules
```
Comment (RomainMuller) [inline packages/jsii/test/negatives/neg.submodules-cannot-share-symbols.ts:1]: Why this change?

Comment (rix0rrr) [reply packages/jsii/test/negatives/neg.submodules-cannot-share-symbols.ts:1]: Because on my machine this was breaking the build. The error message I got said `(ns2 and ns1)` (order different way around), and I did not want to fight with apparent nondeterminism.

Comment (RomainMuller) [reply packages/jsii/test/negatives/neg.submodules-cannot-share-symbols.ts:1]: Ohhhh yeah makes sense!
